src/ComScoreStreamingAnalytics.ts

```diff
@@ -33,6 +34,10 @@ export class ComScoreStreamingAnalytics {
       return;
     }
 
+    if (configuration.userConsent !== undefined) {
```

Comment:
what about `null` handling?

```suggestion
    if (configuration.userConsent != null) { // will handle both cases
```

Reply:
good call, will fix